_infra/helm/locust/locustfiles/collection-exercise-config.json

```diff
@@ -1,5 +1,5 @@
 {
-    "inputFiles": ["/mnt/locust/ce_details.csv"],
+    "inputFiles": [".//ce_details.csv"],
```

Comment:
The existing `/mnt` file location is specific to the containerised helm deployment. The change here is to allow the file to be references from a local non-containerised workspace.